src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/http_client.html.twig

```diff
@@ -85,6 +98,11 @@
                                         <td>
                                             {{ profiler_dump(trace.info, maxDepth=1) }}
                                         </td>
+                                        {% if profiler_token %}
+                                            <td>
+                                                <span><a href="{{ path('_profiler', {'token': profiler_token}) }}">{{ profiler_token }}</a></span>
```

Comment:
Can't the token point to another Symfony app? This doesn't have to be a profile for the current app!